andela/fargo-ah
# #159593957 User Can Pay For Premium Article

##### Note: This feature depends on code from some commits that haven't been merged.
#### What does this PR do?
- Allows users to pay for articles
#### Description of Task to be completed?
- setup payment processor
- setup db for payment
- grant user article access on payment success
- write unit tests for features
#### How should this be manually tested?
In postman:
- To pay for an article, point your Stripe payment form's action is set to: `http://localhost:3000/api/pay/:articleSlug` and also make sure there's a hidden input tag containing authentication token in the form body.
#### Any background context you want to provide?
- Pull in changes and run `npm install`
- Start server with `npm run dev`
- Already created an account through `http://localhost:3000/api/users`
- A Premium article already created
- A frontend payment form already exists to interact with the endpoint
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/159593957
